tensorboard/util.py

```diff
@@ -0,0 +1,224 @@
+# Copyright 2017 The TensorFlow Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""TensorBoard helper routine module.
```

Comment:
What do you think about renaming this to `logging_util.py`?

These routines won't be used anywhere else than their current call sites in `main.py`, so it doesn't seem appropriate for them to be in a module of "generic helper routines." I've found that a good rule of thumb is to only move something to `util.py` once it's been used about five times throughout the codebase. The only thing that seems like it _might_ be reused here is `Ansi`, which we can migrate to a `util.py` if it indeed becomes commonly used.